governance-proposals/term-1/ep12-working-group-rules.md

```diff
@@ -106,7 +102,11 @@ The specification below also introduces a rule to appoint a new DAO Secretary, r
     1. Stewards are eligible to receive fair compensation for their work as a Steward or Lead Steward in the DAO.
     2. All requests for Steward or Lead Steward compensation must be detailed in a Collective Proposal for working group funds submitted to the DAO in accordance with rule 10.1 above.
     3. Stewards may not receive compensation for their role as a Steward or Lead Steward outside of that compensation expressly provided for in a Collective Proposal submitted to the DAO in accordance with rule 10.1 above.
-12. Amendments
```

Comment:
### Suggested Changes
 4.  The Meta-Governance working group are responsible for defining standards for fair compensation ('Compensation Guidelines').
    5. The Compensation Guidelines shall be defined prior to the Nomination Window for each term.
    
### Purpose of Changes
 4.  (This establishes "Compensation Guidelines" as formal terminology in this document and give MG working authority over them)
    5. (Establishes requirement to have compensation guidelines defined BEFORE steward elections. Adding predictability for nominees and uses official terminology elsewhere from document.)

Reply:
Is the pre-term guidance on compensation meant to be binding?  Would this indicate that steward compensation couldn’t be changed during term using the collective proposal process outlined in 10.1, as was allowed previously?
A year is a long time, and things can change in that period that would require deviations.  It’s not clear to me if that would be allowed under these revisions.